client/my-sites/themes/controller.js

```diff
@@ -130,7 +131,16 @@ export function details( context, next ) {
 		isFullScreen: true
 	} ) );
 
-	context.primary = makeElement( ThemeSheetComponent, Head, context.store, props );
 	context.secondary = null; // When we're logged in, we need to remove the sidebar.
+	//TODO: use makeElement()
+	context.primary = (
+		<ReduxProvider store={ context.store } >
+			<Head title={ props.title } isSheet>
```

Comment:
Yeah, I think we should leave this until after routing. It's possibly being implemented in one of those PRs.